examples/year_playlist.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+
+"""Script for retrieving songs from a user's 'Your Top Songs 2017'
+playlist.
+
+Note: Requests to Spotify API do not currently include relevant fields arg,
+which should eventually be added.
+"""
+
+from collections import namedtuple
+import sys
+import argparse
+import spotipy
+from spotipy import oauth2
+
+# Client ID and Client Secret for zachthehammer's spotipy-test API key
+_CLIENT_ID     = '5c243ed8edc9475a90bda2cc4a9a828c'
```

Comment:
style request: let's not keep the value of the constants in line with each other. this always seems like extra unneeded work to me. I prefer: 

```py
_CLIENT_ID = '5c243ed8edc9475a90bda2cc4a9a828c'
_CLIENT_SECRET = 'b414caddc10143eea70db6edc1f4fb47

Reply:
👍